### DISCUSSION TITLE
Which old drivers in general do the old Windows support as of today?

### DISCUSSION THREAD

**Discussion by BridgeHeadland** [joncampbell123/dosbox-x#4554] [Q&A] [answered]
I see that the old Windows, up to XP (or Server 2003, if DOSBox-X can run it) currently do not support all the old drivers and such, I have tried to install Diamond drivers, both sound and graphics drivers, in Windows ME, with no luck, so I'm wondering if there is anyone here who could respond with a list of all the very latest among the best drivers for each of the old Windows, the ones that can be installed and used, as long as those Windows are running virtual via DOSBox-X, as of today.
I would also appreciate if the list will be updated as DOSBox-X becomes more flexible.

**Comment by rderooy** [comment, marked as answer]
Please see the windows installation guides on the wiki for video driver instructions.

https://dosbox-x.com/wiki/Guide%3AMicrosoft-Windows-in-DOSBox%E2%80%90X
